tasks/src/summarization/about.md

```diff
@@ -0,0 +1,27 @@
+## Use Case: Research Paper Summarization
+Research papers can be summarized to enable researchers spend less time on selecting papers to be read among a list of papers. You can directly read papers through Optical Character Recognition models. Feed the output of the OCR to an already existing extractive summarization model on the Hugging Face Hub and infer, so that you can get the summaries of each paper. You can also take a BERT-like (encoder-only) model from the Hugging Face Hub and fine-tune on biomedical text and then fine-tune on summarization task. You can also use sequence-to-sequence models like T5 for abstractive text summarization.
```

Comment:
Up to you but...

I think we can remove the OCR part of this since it just makes this more complicated. Just mentioning the summarization from a given text is good imo.